File: Python/median-of-two-sorted-arrays.py
from typing import List
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        combined = nums1.extend(nums2)
        nums1 = sorted(nums1)
        size = len(nums1)
        isOdd = size % 2 != 0
        return nums1[int(size/2)] if isOdd else (nums1[int(size/2)] + nums1[int(size/2) - 1])/2 

File: Typescript/median-of-two-sorted-arrays.ts
function findMedianSortedArrays(nums1: number[], nums2: number[]): number {
    const combinedSort = [...nums1, ...nums2].sort((a, b) => a - b);
    const size = combinedSort.length;
    const isOdd:boolean = combinedSort.length % 2 !== 0;
    const median:number = isOdd ? combinedSort[Math.round(size/2 - 1)]: (combinedSort[size/2 - 1] + combinedSort[size/2])/2;
    return median;
};